rust-toolchain

```diff
@@ -0,0 +1,3 @@
+# A regression happened in rust-nightly 1.46.0. Therefore, we set the previous toolchain
+# to compile & run the tests with rust 1.45.0.
+nightly-2020-06-04
```

Comment:
I don't think you can add comment here, only the toolchain name – you can add as commit description for the file.

Nit: remember the new line at the end of the file